code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -67,11 +67,9 @@
 		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM * delta_time)
 		return TRUE
 
-/datum/species/pod/randomize_main_appearance_element(mob/living/carbon/human/human_mob)
-	var/hairstyle = pick(GLOB.pod_hair_list)
-	human_mob.dna.features["pod_hair"] = hairstyle
-	mutant_bodyparts["pod_hair"] = hairstyle
-	human_mob.update_body()
+
+/datum/species/pod/randomize_features(mob/living/carbon/human_mob)
+	randomize_external_organs(human_mob)
```

Comment:
You're not randomizing pod hair here?

Reply:
Pod hair is an external organ.
I hate it too, don't worry.

Reply:
hair in general should be an external organ because its currently hardcoded snowflake magic, but i digress

Reply:
One way or another, it works like this currently.